code/game/machinery/vending/_vending.dm

```diff
@@ -161,24 +186,61 @@
 	/// Sound made when an item is vended
 	var/vending_sound = 'sound/machines/vending/vending_drop.ogg'
 
-	/// Lighting mask for vending machine
-	var/light_mask
-
 	/// This is for scaling the ui buttons - i've settled on 80x80 for machines with prices, and 60x60 for those without and with large inventories (boozeomat)
 	var/ui_size = 80
 	var/datum/asset/spritesheet/vending/v_asset
 
+	// Null/set to 0 for vending machines that shouldn't glow for some reason (Horizon recovery ward)
+	/// Lighting mask for vending machine
+	var/light_mask
 	light_range = 2
 	light_power = 0.9
 
+	/**
+	  * Is this item on station or not
+	  *
+	  * if it doesn't originate from off-station during mapload, all_products_free gets automatically set to TRUE if it was unset previously.
```

Comment:
wording here is a bit funny, also thinking
places like port of call maps make sense to still have vending machines that cost money, those would use an override?